### ISSUE TITLE
Real-time conversion of Dolby Vision Profile 7 to 8.1

### ISSUE THREAD

**Issue by Moxh3** [jellyfin/jellyfin-androidtv#2807]
### Describe the feature you'd like

Most Android TV players do not support Dolby Vision Profile 7. However, a lot of them do support Profile 8.1.

It would be great for the client (or server) to use libdovi (https://github.com/quietvoid/dovi_tool/) to covert Profile 7 to 8.1 in real-time.

**Comment by nielsvanvelzen** [comment]
This so called "Dolby Vision Metadata Parser library" doesn't even exist. Don't trust ChatGPT answers without verifying everything it says.

**Comment by rbjtech-github** [comment]
I have used dovi_tool a lot and I'm not even sure this is technically possible in real-time ?   The tool and all tools based on it are very sequential in their processing of the file... ?

**Comment by Moxh3** [comment]
There are Kodi builds that use libdovi (the library) for real-time conversion. Dovi_tool uses libdovi.

**Comment by rbjtech-github** [comment]
> There are Kodi builds that use libdovi (the library) for real-time conversion. So yes, technically feasible. Dovi_tool uses libdovi.

ok - I didn't know that - thanks for the info.

**Comment by RDLN7** [comment]
I really hope to see this implemented soon. It’s been the biggest issue for me when using the Jellyfin Android TV app, especially since most movies nowadays are encoded in Dolby Vision Profile 7.

**Comment by marcokloeckler** [comment]
Just for future reference, here is the PR where Kodi used libdovi to achieve the real-time conversion: https://github.com/xbmc/xbmc/pull/22546

**Comment by DrHurt** [comment]
I agree this would make a useful addition for the many people who play bluray rips (remuxes)
